libgalois/include/galois/OpLog.h

```diff
@@ -19,6 +19,19 @@ enum class OpTypes {
   kOpEdgePropVal,
 };
 
+const char* const optypes_enum2str[] = {
```

Comment:
- Comment on the line above:

I think
```cpp
enum class OpType {
  kInvalid = 0,
  kNodeAdd,
  kNodeDel,
  ...
}; 
```

might be better. "Op" can be dropped from the values because the only way to refer to them is prefixed with "OpType" anyway. Starting from `= 0` is less surprising. What you wrote will result in the same enum values but so would `enum class { kInvalid, kOpNodeAdd, ....}`.

If these values are to be serialized, you can also consider explicitly numbering all the values to prevent any problems is someone accidentally reordered the values.

Reply:
This is not from this PR. It's some old code from @witchel .

Reply:
I have a small preference for `OpType` and the `enum class` values not having "Op" in their names.

Whether this should be now or later is up to you.